grammar.js

```diff
@@ -201,19 +203,19 @@ module.exports = grammar({
       $._lhs
     ),
 
-    scope_resolution_expression: $ => prec.left(seq(optional($._primary), '::', $.identifier)),
-    element_reference: $ => prec.left(seq($._primary, "[", $._argument_list, "]")),
-    member_access: $ => prec.left(seq($._primary, ".", $.identifier)),
+    scope_resolution_expression: $ => prec.left(1, seq(optional($._primary), '::', $.identifier)),
```

Comment:
It might be and it might be good to eventually make these constants like we do for the other precedence values. For now, I'm finding that it's necessary to keep bumping these around and adding to the grammar for a new part of the language often causes cascading changes to these values.